TextMate/TextMate2.munki.recipe

```diff
@@ -21,10 +21,8 @@ preferences panel.
         <string>apps/TextMate2</string>
         <key>NAME</key>
         <string>TextMate2</string>
-        <key>BRANCH</key>
-        <string>release</string>
-        <key>MACOS</key>
-        <string>10.15</string>
+        <key>PRERELEASE</key>
+        <string></string>
```

Comment:
This isn't necessary to include in this recipe.

Reply:
Sorted 😊 output incoming